### PR TITLE
Minimap update

### PR DESCRIPTION
**Pull request by arianna-smith**
Pull Request (PR) checklist:
- [X] Include a description of what is in this pull request in this message.
- [X] The dockerfile successfully builds to a test target for the user creating the PR. (i.e. `docker build --tag samtools:1.15test --target test docker-builds/samtools/1.15` )
- [X] Directory structure as name of the tool in lower case with special characters removed with a subdirectory of the version number (i.e. `spades/3.12.0/Dockerfile`)
- [X] Create a simple container-specific [README.md](https://github.com/StaPH-B/docker-builds/blob/master/.github/workflow-templates/readme-template.md) in the same directory as the Dockerfile (i.e. `spades/3.12.0/README.md`)
- [X] Dockerfile includes the recommended [LABELS](https://github.com/StaPH-B/docker-builds/blob/master/dockerfile-template/Dockerfile#L8-L18) 
- [X] Main [README.md](https://github.com/StaPH-B/docker-builds/blob/master/README.md) has been updated to include the tool and/or version of the dockerfile(s) in this PR
- [X] [Program_Licenses.md](https://github.com/StaPH-B/docker-builds/blob/master/Program_Licenses.md) contains the tool(s) used in this PR and has been updated for any missing

This pull request is to update minimap2 from 2.25 to 2.26.

### CODE REVIEW

File: minimap2/2.26/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+ apt-get autoclean && rm -rf /var/lib/apt/lists/*
+
+# install minimap2 binary; make /data
+RUN curl -L https://github.com/lh3/minimap2/releases/download/v${MINIMAP2_VER}/minimap2-${MINIMAP2_VER}_x64-linux.tar.bz2 | tar -jxvf - && \
+ mkdir /data
+
+# set final PATH and LC_ALL for singularity compatibility
+ENV PATH="${PATH}:/minimap2-${MINIMAP2_VER}_x64-linux" \
+LC_ALL=C
+
```
Comment (erinyoung) [inline minimap2/2.26/Dockerfile:34]: This looks great!

Can you add a CMD instruction to the app stage? This the default command that is run in the container. For STaPH-B, we like to use the tool's help option.

For minimap2 it'd be something like

```
CMD minimap2 -h
```

**Comment by erinyoung** [comment]
Also, did you want to add your github handle to the bottom of the main README.md? I love it when people get credit for what they work on.

**Comment by erinyoung** [comment]
Closes https://github.com/StaPH-B/docker-builds/issues/736

File: minimap2/2.26/Dockerfile (hunk 1 above)
Comment (arianna-smith) [reply minimap2/2.26/Dockerfile:34]: Done!

**Review by erinyoung — APPROVED** [review]
Perfect! Looks great to me.

I'm going to
- merge this PR
- deploy to dockerhub and quay with the tag '2.26'
- overwrite the 'latest' tag

**Comment by erinyoung** [comment]
Thank you for your contribution!

You can check the status of the deploy at https://github.com/StaPH-B/docker-builds/actions/runs/7038701248

**Pull request merged**